plugins/helper.js

```diff
@@ -0,0 +1,24 @@
+import { inject } from "vue"
+
+import moment from "moment"
+
+const helper = {
+    getCurrentTimeInMS: () => {
+        return Date.now()
+    },
+
+    objectDeepClone: (obj = {}) => {
```

Comment:
please use typescript

Reply:
It is a .js file. I will use .ts files with type script in acutal projects

Reply:
resolved